games/applications/dj.py

```diff
@@ -0,0 +1,97 @@
+"""Deustch Jozsa Benchmark Circuits"""
```

Comment:
This file needs a copyright header, you can see what other source files do and just copy that. (although note the repo is apache licensed some of the other files says MIT).